components/brave_vpn/resources/panel/components/contact-support/index.tsx

```diff
@@ -5,11 +5,12 @@ import TextInput, { Textarea } from '$web-components/input'
 import Toggle from '$web-components/toggle'
 import { getLocale } from '../../../../../common/locale'
 import * as S from './style'
+import { ErrorLabel } from '../styles/style'
 import getPanelBrowserAPI, * as BraveVPN from '../../api/panel_browser_api'
 import { CaratStrongLeftIcon } from 'brave-ui/components/icons'
 
 interface Props {
-  onCloseContactSupport: React.MouseEventHandler<HTMLButtonElement>
```

Comment:
we pass outside only empty callbacks to setup flags, seems not necessary to pass `React.MouseEventHandler` @petemill ? and I added auto closing of this panel to be more user friendly when request is success